### PR TITLE
[AKS] Set network profile when using basic load balancer.

### PR DESCRIPTION
**Pull request by JunSun17**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by JunSun17** [comment]
This is for #2138

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -988,6 +988,10 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
                 load_balancer_profile=load_balancer_profile,
                 outbound_type=outbound_type,
             )
+        if load_balancer_sku.lower() == "basic":
+            network_profile = ContainerServiceNetworkProfile(
+                load_balancer_sku=load_balancer_sku.lower(),
```
Comment (robbiezhang) [inline src/aks-preview/azext_aks_preview/custom.py:993]: please verify whether you need to set network_plugin="kubenet" like L986

Comment (JunSun17) [reply src/aks-preview/azext_aks_preview/custom.py:993]: Verified in live test that network_plugin is not needed, it default to kubenet.

**Comment by yonzhan** [comment]
aks

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to review the PR?

**Pull request merged**